gradle.properties

```diff
@@ -12,15 +12,15 @@ kotlin.version.snapshot=1.3-SNAPSHOT
 # Also set KONAN_LOCAL_DIST enviroment variable in bootstrap mode to auto-assign konan.home
 
 junit_version=4.12
-infra_version=0.1.0-dev-42
+infra_version=0.1.0-dev-47
 bintray_version=1.8.4-jetbrains-5
 benchmarks_version=0.1.7-dev-14
 
 native.deploy=false
 
-kotlin.parallel.tasks.in.project=true
+#kotlin.parallel.tasks.in.project=true
```

Comment:
I think this should be uncommented for local development and disabled via system properties on TeamCity